stripe/lib/handlers/handleCheckoutSession.ts

```diff
@@ -1,8 +1,9 @@
 import Stripe from "stripe";
-import { HandlerReturn } from "..";
-import { CtlxClientType } from "../client";
+import { HandlerReturn } from "../../../utils";
```

Comment:
Nit: You could make the imports prettier using the tsconfig.

Reply:
done